### PR TITLE
ABE-655 - Create a dummy admin service.

### PR DESCRIPTION
**Pull request by michaelleahy**
@joshhaynes @jsatria @lookinginstereo 

Ignore the ABE-653 commit, that'll get sucked up once PR #2 is merged.  You guys are falling behind.

### CODE REVIEW

**Comment by Ackertron** [comment]
retest this please

File: service/src/main/scala/com/rallyhealth/donations/service/DonationsAdminService.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  /**
+   * Get all donation periods in the system.
+   */
+  def getAllDonationPeriods: Future[Seq[DonationOverview]]
+
+  /**
+   * Sends test emails
+   * Probably needs some fields and other ish...
+   */
+  def sendTestEmail: Future[Unit]
```
Comment (lookinginstereo) [inline service/src/main/scala/com/rallyhealth/donations/service/DonationsAdminService.scala]: No longer necessary.

**Comment by Ackertron** [comment]
retest this please

**Comment by Ackertron** [comment]
retest this please

File: service/src/main/scala/com/rallyhealth/donations/service/DonationsAdminService.scala (hunk 1 above)
Comment (michaelleahy) [inline service/src/main/scala/com/rallyhealth/donations/service/DonationsAdminService.scala]: Fixed in next PR.

**Comment by joshhaynes** [comment]
LGTM

**Comment by michaelleahy** [comment]
1LZF

**Pull request merged**